### PR TITLE
Feat/new transaction filter

### PR DESCRIPTION
**Pull request by ratik**
https://p2pvalidator.atlassian.net/browse/NTRN-262

### CODE REVIEW

**Review by pr0n00gler — APPROVED** [review]

**Comment by pr0n00gler** [comment]
Are there any scripts/tests to check this actually works?

**Comment by swelf19** [comment]
We could check, the new query data corresponds the query type

**Comment by ratik** [comment]
> We could check, the new query data corresponds the query type

I'm afraid we don't have info what kind of query we updating in this case so we can't check it this way

**Comment by ratik** [comment]
> Are there any scripts/tests to check this actually works?

not yet, will do

**Review by swelf19 — APPROVED** [review]

**Review by sotnikov-s — COMMENTED** [review]
looks good. what about integration tests for this feature?

**upd:** ignore — I can see the comment above. waiting for the test before I approve

**Review by quasisamurai — APPROVED** [review]

File: packages/neutron-sdk/src/bindings/msg.rs
Comment (sotnikov-s) [inline packages/neutron-sdk/src/bindings/msg.rs:209]: do we really need to validate filters len here? neutron icq module does it, why should we do it here? if the max possible value changes in the neutron, we'll have to take care of it here as well

Comment (ratik) [reply packages/neutron-sdk/src/bindings/msg.rs:209]: MAX_TX_FILTERS has been introduced not in this PR so I believe this case is out of scope of this PR.
Anyway I don't see any issue about handling this value in neutron and here, bc we already have to handle really many things like message formats (bindings) and etc

**Review by sotnikov-s — APPROVED** [review]

**Review by oopcode — APPROVED** [review]

**Pull request merged**
